tests/tests_twobody/test_propagation.py

```diff
@@ -390,6 +391,19 @@ def test_long_propagations_vallado_agrees_farnocchia():
     assert_quantity_allclose(v_mm, v_k)
 
 
+def test_farnocchia_propagation_very_high_ecc_does_not_fail():
+    # Regression test for #1296.
+    r = np.array([-500, 1500, 4012.09]) << u.km
+    v = np.array([5021.38, -2900.7, 1000.354]) << u.km / u.s
+    orbit = Orbit.from_vectors(Earth, r, v, epoch=time.Time("2020-01-01"))
+
+    tofs = [74] << u.s  # tof = 74s and above is the critical region.
+    coords = propagate(orbit, tofs)
+
+    assert not np.isnan(coords[0].get_xyz()).any()
+    assert not np.isnan(coords[1].get_xyz()).any()
```

Comment:
These lines are failing. Maybe they're not needed?

```suggestion
    assert not np.isnan(coords.get_xyz()).any()
```

Reply:
(Haven't tested this, not sure if it's enough)

Reply:
Indeed, that works! Thanks...